lib/src/main/kotlin/com/swmansion/starknet/data/serializers/BlockIdSerializer.kt

```diff
@@ -0,0 +1,41 @@
+package com.swmansion.starknet.data.serializers
+
+import com.swmansion.starknet.data.types.BlockId
+import com.swmansion.starknet.data.types.BlockTag
+import com.swmansion.starknet.data.types.Felt
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.json.JsonEncoder
+import kotlinx.serialization.json.buildJsonObject
+import kotlinx.serialization.json.encodeToJsonElement
+import kotlinx.serialization.json.put
+
+internal object BlockIdSerializer : KSerializer<BlockId> {
```

Comment:
JvmSynthetic for internals?

Reply:
You cannot have this annotation on classes/objects unfortunately.